R/remote.R

```diff
@@ -58,44 +58,55 @@ users_create_remote <- function(connect, prefix, expect = 1, check = TRUE, exact
     src = connect
   )
   message("Done creating remote users")
-  return(get_users(connect, prefix = prefix))
+  return(user_creation)
```

Comment:
Should this be reverted?

Reply:
Good catch, I only reverted the groups one.